apps/web/Dockerfile

```diff
@@ -1,11 +1,77 @@
 FROM node:18-alpine AS installer
 RUN corepack enable && corepack prepare pnpm@latest --activate
 
+ARG NEXT_PUBLIC_PRIVACY_URL
+ENV NEXT_PUBLIC_PRIVACY_URL=$NEXT_PUBLIC_PRIVACY_URL
+
+ARG NEXT_PUBLIC_TERMS_URL
+ENV NEXT_PUBLIC_TERMS_URL=$NEXT_PUBLIC_TERMS_URL
+
+ARG NEXT_PUBLIC_IMPRINT_URL
+ENV NEXT_PUBLIC_IMPRINT_URL=$NEXT_PUBLIC_IMPRINT_URL
+
+ARG NEXT_PUBLIC_WEBAPP_URL
+ENV NEXT_PUBLIC_WEBAPP_URL=$NEXT_PUBLIC_WEBAPP_URL
+
+ARG NEXT_PUBLIC_SURVEY_BASE_URL
+ENV NEXT_PUBLIC_SURVEY_BASE_URL=$NEXT_PUBLIC_SURVEY_BASE_URL
+
+ARG NEXT_PUBLIC_EMAIL_VERIFICATION_DISABLED
+ENV NEXT_PUBLIC_EMAIL_VERIFICATION_DISABLED=$NEXT_PUBLIC_EMAIL_VERIFICATION_DISABLED
+
+ARG NEXT_PUBLIC_PASSWORD_RESET_DISABLED
+ENV NEXT_PUBLIC_PASSWORD_RESET_DISABLED=$NEXT_PUBLIC_PASSWORD_RESET_DISABLED
+
+ARG NEXT_PUBLIC_SIGNUP_DISABLED
+ENV NEXT_PUBLIC_SIGNUP_DISABLED=$NEXT_PUBLIC_SIGNUP_DISABLED
+
+ARG NEXT_PUBLIC_INVITE_DISABLED
+ENV NEXT_PUBLIC_INVITE_DISABLED=$NEXT_PUBLIC_INVITE_DISABLED
+
+ARG WEBAPP_URL
+ENV WEBAPP_URL=$WEBAPP_URL
+
+ARG DATABASE_URL
+ENV DATABASE_URL=$DATABASE_URL
+
+ARG NEXTAUTH_SECRET
+ENV NEXTAUTH_SECRET=$NEXTAUTH_SECRET
+
+ARG NEXTAUTH_URL
+ENV NEXTAUTH_URL=$NEXTAUTH_URL
+
+ARG SENTRY_IGNORE_API_RESOLUTION_ERROR
+ENV SENTRY_IGNORE_API_RESOLUTION_ERROR=$SENTRY_IGNORE_API_RESOLUTION_ERROR
+
+ARG NEXT_PUBLIC_SENTRY_DSN
+ENV NEXT_PUBLIC_SENTRY_DSN=$NEXT_PUBLIC_SENTRY_DSN
+
+ARG NEXT_PUBLIC_GITHUB_AUTH_ENABLED
+ENV NEXT_PUBLIC_GITHUB_AUTH_ENABLED=$NEXT_PUBLIC_GITHUB_AUTH_ENABLED
+
+ARG GITHUB_ID
+ENV GITHUB_ID=$GITHUB_ID
+
+ARG GITHUB_SECRET
+ENV GITHUB_SECRET=$GITHUB_SECRET
+
+ARG NEXT_PUBLIC_GOOGLE_AUTH_ENABLED
+ENV NEXT_PUBLIC_GOOGLE_AUTH_ENABLED=$NEXT_PUBLIC_GOOGLE_AUTH_ENABLED
+
+ARG GOOGLE_CLIENT_ID
+ENV GOOGLE_CLIENT_ID=$GOOGLE_CLIENT_ID
+
+ARG GOOGLE_CLIENT_SECRET
+ENV GOOGLE_CLIENT_SECRET=$GOOGLE_CLIENT_SECRET
+
+ARG CRON_SECRET
+ENV CRON_SECRET=$CRON_SECRET
+
 WORKDIR /app
 
 COPY . .
-# Copy .env file because Docker don't follow symlinks
-COPY .env.docker /app/apps/web/.env
+RUN touch /app/apps/web/.env
```

Comment:
why is this command needed? Shouldn't it also work with no `.env` file?

Reply:
I tried :/, it doesn't work ;( hence passed this empty env